### ISSUE TITLE
No Polygon (area) support

### ISSUE THREAD

**Issue by alexgleith** [NathanW2/MapInfo-to-QGIS-style-generator#2]
I wish this did work with polygons, I know it'll be a pain in the arse, Nathan, but I know you can do it!

**Comment by NathanW2** [comment]
Depends on how much you are willing to pay me to open MapInfo again ;)

**Comment by alexgleith** [comment]
Yep, I was joking, and I know that it would be really painful to do. So ignore this... =)
